docs/pages/_app.js

```diff
@@ -368,9 +369,11 @@ export default function MyApp(props) {
   const getLayout = Component.getLayout ?? ((page) => page);
 
   return (
-    <AppWrapper emotionCache={emotionCache} pageProps={pageProps}>
-      {getLayout(<Component {...pageProps} />)}
-    </AppWrapper>
+    <CacheProvider value={emotionCache}>
```

Comment:
This reminds me of another deeper problem. Why do we expose `@mui/material-nextjs` to users if we don't fully use it on the docs (we only use documentGetInitialProps from it, but not AppCacheProvider)? We should move to dogfood it?

(*Arguably, it's a `@mui/system-nexjs` module but that's another problem*)